bedrock/firefox/templates/firefox/releases/notes.html

```diff
@@ -394,10 +394,22 @@ <h2>Other Resources</h2>
 {% block email_form %}{% endblock %}
 
 {% macro note_entry(note) %}
-  <li id="note-{{ note.id }}">
-    {{ note.note|safe }}
-    {% if release.channel == 'Nightly' and note.bug -%}
-    <span class="bug-id"><a href="https://bugzilla.mozilla.org/show_bug.cgi?id={{ note.bug }}">Bug {{ note.bug }}</a></span>
-    {%- endif %}
+  <li class="release-note" id="note-{{ note.id }}">
+    <div class="release-note-content">
+      {{ note.note|safe }}
+      {% if release.channel == 'Nightly' and note.bug -%}
+      <span class="bug-id"><a href="https://bugzilla.mozilla.org/show_bug.cgi?id={{ note.bug }}">Bug {{ note.bug }}</a></span>
+      {%- endif %}
+    </div>
+    {% if note.progressive_rollout %}
+    <div class="release-note-progressive-rollout-indicator">
+      <img src="{{ static('img/firefox/releasenotes/progressive.svg') }}" alt="">
```

Comment:
Suggestion: add `loading="lazy"`
```suggestion
      <img loading="lazy" src="{{ static('img/firefox/releasenotes/progressive.svg') }}" alt="">
```